op-node/rollup/derive/attributes_queue.go

```diff
@@ -33,6 +36,40 @@ type AttributesWithParent struct {
 	IsLastInSpan bool
 }
 
+func (a *AttributesWithParent) ToBuilderPayloadAttributes() *BuilderPayloadAttributes {
+	transactions := make([]*types.Transaction, len(a.Attributes.Transactions))
+	for i, txBytes := range a.Attributes.Transactions {
+		var ttx types.Transaction
+		ttx.UnmarshalBinary(txBytes)
```

Comment:
I think this can return an error, good to log something here at least

Reply:
fixed to return error properly fd070617a07a24d0148519616b22717d5e450114